### DISCUSSION TITLE
Can I build this project with a mac

### DISCUSSION THREAD

**Discussion by besi** [akopachov/flipper-zero_authenticator#222] [Q&A]
In the FAQ it's mentioned that I have to run some power-shell script to build the source. So I'm curious. Is it possible to build this on Unix or Mac?

**Comment by akopachov** [comment]
Sure, it is definitely possible. That Powershell scripts mentioned in FAQ are doing nothing more then just simplifying and automating builds steps, but internally they rely on official [uFBT](https://github.com/flipperdevices/flipperzero-ufbt) build tool, which is python-based and cross-platform. So you may just install and use uFBT manually without any Powershell script.
